### DISCUSSION TITLE
Is there any Ansible role to install NHC?

### DISCUSSION THREAD

**Discussion by btravouillon** [mej/nhc#148] [Q&A]
From https://github.com/mej/nhc/issues/138#issuecomment-1652292394

> Here at LANL, though, we currently use Ansible for doing config management, and our NHC role supplies the desired checks for each context as a named YAML list and a Jinja2 template that iterates over them.

By any chance, is this role available publicly?

We need to rewrite our current Ansible role to install and setup NHC and I would prefer to use any community role instead of reinventing the wheel.

**Comment by jebbaxley** [comment]
I can help write the ansible role if this is still needed.  Sent from my iPhoneOn Dec 7, 2023, at 11:51, Bruno Travouillon ***@***.***> wrote:﻿
From #138 (comment)

Here at LANL, though, we currently use Ansible for doing config management, and our NHC role supplies the desired checks for each context as a named YAML list and a Jinja2 template that iterates over them.

By any chance, is this role available publicly?
We need to rewrite our current Ansible role to install and setup NHC and I would prefer to use any community role instead of reinventing the wheel.
